tox.ini

```diff
@@ -1,18 +1,8 @@
 [tox]
-skipsdist = True
-envlist = unit, flake8, integration-sqlserver, integration-synapse
+envlist = py37
 
-[testenv:integration-sqlserver]
-basepython = python3
-commands = /bin/bash -c '{envpython} -m pytest -v test/integration/sqlserver.dbtspec'
+[testenv]
+commands = pytest {posargs}
 deps =
-    -r{toxinidir}/requirements.txt
-    -e.
-
-[testenv:integration-azuresql]
-basepython = python3
-commands = /bin/bash -c '{envpython} -m pytest -v test/integration/azuresql.dbtspec'
-passenv = DBT_AZURESQL_DB DBT_AZURESQL_PWD DBT_AZURESQL_SERVER DBT_AZURESQL_UID
-deps =
-    -r{toxinidir}/requirements.txt
-    -e.
\ No newline at end of file
+  pytest~=6.2.2
+  pytest-dbt-adapter~=0.4.0
```

Comment:
1. Since we are only running it for one python version we could simply call pytest directly.
2. This is a tricky one. In general, I think there is benefit for consistency, it is easier for collaborating. However, I feel that it is "off" to create a copy of the test environment for each subset of test we would like to run, while you can use `{posargs}`. Resulting in that if you want to change the test environment in tox, you have to change it in many places (although a search an replace might solve that). I copied this mostly from the `tox` repo and docs.
3. See this [`tox.ini`](https://github.com/tox-dev/tox/blob/master/tox.ini), maybe for linting and for docs generation. But I think the "real" benefit is testing it against different python versions and for different OSs.